data-serving/scripts/data-pipeline/convert_and_import_latest_data.sh

```diff
@@ -87,21 +88,27 @@ function cleanup() {
 
 # Establish run order
 main() {
-    print "Running with:\n
-    [-m] MongoDB connection string: $mongodb_connection_string
-    [-d] database: $db
-    [-c] collection: $collection
-    [-r] sample rate: $sample_rate
-    [-s] schema path: $schema_path
-    [-i] indexes path: $indexes_path"
+    if [ -z "${source_id}"  ]
+    then   
+        print "-e flag for source_id is required"
```

Comment:
exit -1 if required param is missing instead of indenting everything after that in an else?

Reply:
Done